main.go

```diff
@@ -244,14 +244,18 @@ func setupScheduler(mgr ctrl.Manager, cCache *cache.Cache, queues *queue.Manager
 		cCache,
 		mgr.GetClient(),
 		mgr.GetEventRecorderFor(constants.AdmissionName),
-		scheduler.WithWaitForPodsReady(waitForPodsReady(cfg)),
+		scheduler.WithBlockForPodsReady(blockForPodsReady(cfg)),
 	)
 	if err := mgr.Add(sched); err != nil {
 		setupLog.Error(err, "Unable to add scheduler to manager")
 		os.Exit(1)
 	}
 }
 
+func blockForPodsReady(cfg *config.Configuration) bool {
```

Comment:
nit: IIRC we could use this function when creating the cache (the `WithPodsReadyTracking` option), as the cache is used only for blocking, not for enforcing timeout.

Reply:
Done~ 😄